tests/h/services/search_index/_queue_test.py

```diff
@@ -148,36 +155,28 @@ def test_it_does_nothing_if_the_queue_is_empty(self, batch_indexer, queue):
         batch_indexer.index.assert_not_called()
 
     def test_it_ignores_jobs_that_arent_scheduled_yet(
-        self, annotation_ids, batch_indexer, queue
+        self, add_all, batch_indexer, queue
     ):
-        queue.add_all(annotation_ids, tag="test_tag", schedule_in=ONE_WEEK_IN_SECONDS)
+        add_all(schedule_in=ONE_WEEK_IN_SECONDS)
 
         queue.sync(LIMIT)
 
         batch_indexer.index.assert_not_called()
 
+    @pytest.mark.usefixtures("with_queued_annotations")
     def test_it_ignores_jobs_beyond_limit(
         self, all_annotation_ids, batch_indexer, queue
     ):
-        queue.add_all(
-            all_annotation_ids, tag="test_tag", schedule_in=MINUS_5_MIN_IN_SECS
-        )
```

Comment:
Lots of these were all the same, so I packed this all away in the fixture. I think this leaves "what's important" a bit clearer. For example the "test_tag" here is required to make the call, but doesn't have any test relevance.